docs/language-usage.rst

```diff
@@ -0,0 +1,354 @@
+:attr:`Document.HTML <gcloud.language.document.Document.HTML>`.
+
+In addition to supplying the text / HTML content, a document can refer
+to content stored in `Google Cloud Storage`_:
+
+  .. code-block:: python
+
+     >>> document = client.document(bucket='my-text-bucket',
+     ...                            blob='sentiment-me.txt')
+     >>> document.gcs_url
+     'gs://my-text-bucket/sentiment-me.txt'
+
+.. _analyzeEntities: https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/analyzeEntities
+.. _analyzeSentiment: https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/analyzeSentiment
+.. _annotateText: https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/annotateText
+.. _Document: https://cloud.google.com/natural-language/reference/rest/v1beta1/Document
+.. _Google Cloud Storage: https://cloud.google.com/storage/
+
+Analyze Entities
+----------------
+
+The :meth:`~gcloud.language.document.Document.analyze_entities` method
+finds named entities (i.e. proper names) in the text, entity types,
+salience (prominence), mentions for each entity, and other properties.
+
+  .. code-block:: python
+
+     >>> text_val = ("Michelangelo Caravaggio, Italian painter, is "
+     ...             "known for 'The Calling of Saint Matthew'.")
+     >>> document = client.document(text_val)
+     >>> document.analyze_entities()
+     {
+       'entities': [
+         {
+           'name': 'Michelangelo Caravaggio',
+           'type': 'PERSON',
+           'metadata': {
+             'wikipedia_url': 'http://en.wikipedia.org/wiki/Caravaggio'
+           },
+           'salience': 0.75942981,
+           'mentions': [
+             {
+               'text': {
+                 'content': 'Michelangelo Caravaggio',
+                 'beginOffset': 0
+               }
+             }
+           ]
+         },
+         {
+           'name': 'Italian',
+           'type': 'LOCATION',
+           'metadata': {
+             'wikipedia_url': 'http://en.wikipedia.org/wiki/Italy'
+           },
+           'salience': 0.20193423,
+           'mentions': [
+             {
+               'text': {
+                 'content': 'Italian',
+                 'beginOffset': 25
+               }
+             }
+           ]
+         },
+         {
+           'name': 'The Calling of Saint Matthew',
+           'type': 'WORK_OF_ART',
+           'metadata': {
+             'wikipedia_url': 'http://en.wikipedia.org/wiki/index.html?curid=2838808'
+           },
+           'salience': 0.03863598,
+           'mentions': [
+             {
+               'text': {
+                 'content': 'The Calling of Saint Matthew',
+                 'beginOffset': 56
+               }
+             }
+           ]
+         }
+       ],
+       'language': 'en'
+     }
+
+In addition, the encoding of the text in the document can be
+specified in the request. By default, the UTF-8 is used:
```

Comment:
But yes - you're correct in that if you get an arbitrary byte string from the user, utf8 is as sensible a default as any, since it's so ubiquitous. But if you get an arbitrary `unicode` string (or if you're working in python3, where I believe everything is a `unicode` by default), then it makes sense to default to UTF16/32.